src/commands/coverage/upload.ts

```diff
@@ -58,6 +78,10 @@ export class UploadCodeCoverageReportCommand extends Command {
     `,
     examples: [
       ['Upload all code coverage report files in current directory', 'datadog-ci coverage upload .'],
```

Comment:
would it be current directory or current directory and all the subfolders?

Reply:
Subfolders are included. Added it to the description